docs/tools/terraform/reference/cookbook/singlecloud-postgresql-recipe.rst

```diff
@@ -0,0 +1,62 @@
+Deploy PostgreSQL® service to a single cloud and region
+=========================================
+
+This example shows the setup for a Terraform project containing a single PostgreSQL service. The following setup shows that the Aiven Terraform Provider calls the Aiven API under the hood to create a PostgreSQL service with startup-4 plan on Google Cloud Platform (Europe).
+
+
+Describe the setup
+'''''''''''''''''''''''''''''''''''
+
+Your Terraform files declare the structure of your infrastructure as well as required dependencies and configuration. While you can stuff these together in one file, it's ideal to keep those as separate files.
+
+The following Terraform script deploys a single-node PostgreSQL service. This is a minimal example which you can swap out with your own Terraform scripts or other advanced recipes from :doc:`the Terraform cookbook <reference/cookbook>`.
+
+The contents of the ``postgresql.tf`` file should look like this:
+
+.. code:: bash
+
+  # A single-node PostgreSQL service
+  
+  resource "aiven_pg" "pg" {
+    project                 = var.project_name
+    cloud_name              = "google-europe-west1"
+    plan                    = "startup-4"
+    service_name            = "my-pg1-gcp-eu"
+    maintenance_window_dow  = "monday"
+    maintenance_window_time = "10:00:00"
+    termination_protection  = true  
+
+    pg_user_config {
+      pg_version = 14
+
+    public_access {
```

Comment:
It's great to see public_access and other pg configurations. Let's describe what these configurations do in terms of a use case (or a story). 

Since we already talked about termination_protection in a previous recipe, we can omit that settings here. Our goal is to cover different configurations across various recipes so that the cookbook becomes complete with all the combinations of recipes.

Reply:
I've added some other configurations such as read-replica and custom username and password.